pyccel/ast/datatypes.py

```diff
@@ -358,6 +358,39 @@ def __and__(self, other):
         else:
             return NotImplemented
 
+    @property
+    def min_value(self):
+        """
+        Get the minimum representable value for this datatype.
+
+        This property returns the smallest value that can be represented by
+        this numeric datatype. It is useful for validation, boundary checks,
+        and ensuring that generated code respects the limits of the type.
+
+        Returns
+        -------
+        int
+            The minimum value that can be represented.
+        """
+        return numpy.iinfo(int).min
```

Comment:
This new property needs documenting. Given the urgency of this PR I think it is ok if you create an issue and add the documentation in a second PR